app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
```

Comment:
- [ ] comment, post, and like should belong to the user who owns them. It's why I'll suggest you implement all the `belongs_to :user` as here: `belongs_to :author, class_name: 'User'`.

- [ ] Make sure that you create migration files for the author with foreign key `author_id` by adding references for posts, likes, and comments.

Reply:
And also on this side, since it `belongs_to :user`, which is the same as the class name, I don't explicitly have to tell it the class name
@RolandM99